_data/harnesses/kafka-consumer-application/confluent.yml

```diff
@@ -0,0 +1,171 @@
+dev:
+  steps:
+    - title: Initialize the project
+      content:
+        - action: execute
+          file: tutorial-steps/dev/init.sh
+          render:
+            file: tutorials/kafka-consumer-application/kafka/markup/dev/init.adoc
+
+    - title: Sign up for Confluent Cloud and Get the CLI
+      content:
+        - action: skip
+          render:
+            file: shared/markup/ccloud/get-ccloud.adoc
+
+    - title: Provision resources in Confluent Cloud
+      content:
+        - action: skip
+          render:
+            file: shared/markup/ccloud/ccloud-setup-self.adoc
+
+    - title: Create a topic
+      content:
+        - action: skip
+          render:
+            file: tutorials/kafka-consumer-application/confluent/markup/dev/make-topic.adoc
+
+    - title: Configure the project
+      content:
+        - action: make_file
+          file: build.gradle
+          render:
+            file: tutorials/kafka-consumer-application/kafka/markup/dev/make-build-file.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/gradle-wrapper.sh
+          render:
+            file: tutorials/kafka-consumer-application/kafka/markup/dev/make-gradle-wrapper.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/make-configuration-dir.sh
+          render:
+            file: tutorials/kafka-consumer-application/kafka/markup/dev/make-config-dir.adoc
+
+        - action: skip
+          render:
+            file: shared/markup/ccloud/config-create.adoc
```

Comment:
Perhaps this step that gets the ccloud config should be at the top after the in-product Learn step?  And it could be called ccloud.properties instead of dev.properties, so it can be reused by other tutorials.  i.e., a person who creates 1 ccloud env has one ccloud.properties file, and they have multiple dev.properties, one per tutorial?

Reply:
I gave it a try in https://github.com/confluentinc/kafka-tutorials/pull/786/commits/e8e2214c20c9f7306ef61ccab0415dd765d2b8b0.